apps/styleguide/styleguidist-setup.js

```diff
@@ -0,0 +1,22 @@
+import Vue from "vue";
```

Comment:
I don't want users of styleguide to assume they have router. So why do you need this?

Reply:
'don't want users of styleguide to assume they have router' why not ? we use it everywhere

Reply:
we actually do not afaik. Should double check because maybe by now that has changed. But we don't need it right? So why add this dep then here?